### DISCUSSION TITLE
ORA-01882: timezone region not found

### DISCUSSION THREAD

**Discussion by kabeza** [dbeaver/dbeaver#20912] [General]
Hi
I've installed latest dbeaver-ce using snapd (Ubuntu 22.04, Elementary OS Horus)

`snap install dbeaver-ce`

I want to add a new Oracle connection, but I'm getting the following error (drivers are downloaded correctly, etc.)

![imagen_1537](https://github.com/dbeaver/dbeaver/assets/274856/c0a6055b-42f4-492c-9f2f-1f4660fc3e86)

```
ORA-00604: error ocurred at recursive SQL level 1
ORA-01882: timezone region not found

```

I've seen most solutions refer to adding this line
`-Duser.timezone=UTC-3`
to the dbeaver.ini file. But as I've installed dbeaver-ce with snap, I cannot find dbeaver.ini. Not even in 
`~/snap/dbeaver-ce/248/.local/share`

Does any1 know if the error is related to this? And how could I solve it in Ubuntu?

Thanks

**Comment by LonwoLonwo** [comment]
Hello @kabeza 

Could you please try to add your parameter on the end of the _Exec_ line in /var/lib/snapd/desktop/applications/dbeaver-ce_dbeaver-ce.desktop?

**Reply by kabeza** [reply]
Hi Anastasiya
I've edited the .desktop file. It looks like this:
![imagen_1546](https://github.com/dbeaver/dbeaver/assets/274856/a9b2bd1f-3a1b-46f5-b7e7-a677afffe523)

But I keep getting the same error when trying to connect to that Oracle Server

**Comment by GROL** [comment]
I had the same issue. I fixed it as follows:

- In the main menu, go to Window -> Preferences.
- In the dialog User Interface -> Timezone.
- Set your zone (usually, this value as **default**).

**Reply by kabeza** [reply]
Great!
PS: I've noticed I can't change timezone. I always get it set to default even after restarting

**Reply by AndrewKHML** [reply]
Many Thanks to you!
